install/installation-kubernetes.md

```diff
@@ -0,0 +1,179 @@
+1.  Add the TimescaleDB Helm chart repository:
+    ```bash
+    helm repo add timescale 'https://charts.timescale.com'
+    ```
+1.  Verify that the repository is up to date:
+    ```bash
+    helm repo update
+    ```
+1.  Install the TimescaleDB Helm chart, by replacing `<my_name>` with a name of
+    your choice:
+    ```bash
+    helm install <my_name> timescale/timescaledb-single
+    ```
+    If you created a `<myvalues.yaml>` file, use this command instead:
+    ```bash
+    helm install <my_name> -f <myvalues.yaml> charts/timescaledb-single
+    ```
+
+</procedure> 
+
+## Connect to TimescaleDB
+You can connect to TimescaleDB from an external IP address, or from within the
+cluster.
+
+<procedure>
+
+### Connecting to TimescaleDB using an external IP
+
+<highlight type="note">
+If you configured the user credentials in the `my_values.yaml` file, you don't
+need to decode the passwords. In the following section replace `<my_name>` with
+the name that you provided during the installation.
+</highlight>
+
+1. Get the name of the host to connect to:
+    ```bash
+    kubectl get service/<my_name>
+    ```
+1. Decode the `admin` user password `PGPASSWORD_ADMIN` that was generated during
+   the Helm installation:
+    ```bash
+    PGPASSWORD_ADMIN=$(kubectl get secret --namespace default 
+    <my_name>-credentials -o jsonpath="{.data.PATRONI_admin_PASSWORD}" | base64 --decode)
+    ``` 
+1. **OPTIONAL** Decode the super user password `PGPOSTGRESPASSWORD` that was
+   generated during the Helm installation:
+    ```bash
+    PGPASSWORD_POSTGRES=$(kubectl get secret --namespace default 
+    <my_name>-credentials -o jsonpath="{.data.PATRONI_SUPERUSER_PASSWORD}" | base64 --decode)
+    ```
+1. Connect with psql as `admin` user:
+    ```bash
+    kubectl run -i --tty --rm psql --image=postgres \
+      --env "PGPASSWORD=$PGPASSWORD_ADMIN" \
+      --command -- psql -U admin \
+      -h <my_name>.default.svc.cluster.local postgres
+    ```
+    
+</procedure>
+
+<procedure>
+
+### Connecting to TimescaleDB from inside the cluster
+1. Get the Pod on which TimescaleDB is installed:
+   ```bash
+    MASTERPOD="$(kubectl get pod -o name --namespace default -l release=test,role=master)"
+    ```
+1. Run `psql` inside the Pod containing the primary:
+    ```bash
+    kubectl exec -i --tty --namespace default ${MASTERPOD} -- psql -U postgres
+    ```
+
+</procedure>
+
+<procedure>
+
+## Create a database
+ After installing and connecting to TimescaleDB you can create a database,
+ connect to the database, and also verify that the TimescaleDB extension is
+ installed.
+
+### Creating a database
+1.  At the prompt, create an empty database. For example, to create a database
+    called `tsdb`:
+    ```sql
+    CREATE database tsdb;
+    ```
+1.  Connect to the database you created:
+    ```sql
+    \c tsdb
+    ```
+1. Verify that the TimescaleDB extension is installed by using the `\dx`
+command at the command prompt. It looks like this:
```

Comment:
Fix indents here, the code block is out of alignment:

![image](https://user-images.githubusercontent.com/3914967/157573916-5069d14c-a025-4d32-9e8e-508b5d5cf8de.png)

Reply:
fixed the alignment issue.